src/renderer/components/editorWithTabs/sourceCode.vue

```diff
@@ -164,4 +164,7 @@
   .dark.source-code .CodeMirror-activeline-gutter {
     background: #333;
   }
+  .CodeMirror {
+    height: 100%;
```

Comment:
I think you mean `height: calc(100vh - (var(--titleBarHeight) + 100px));` (100% - titlebar and padding)

Reply:
fixed